qcelemental/molparse/to_string.py

```diff
@@ -136,6 +138,26 @@ def to_dict(self) -> Dict:
 
         smol.extend(atoms)
 
+    if dtype == "orca":
+        # Notes
+        # * if units not in umap (e.g., nm), can't be read back in by from_string()
+
+        atom_format = "{elem}" if atom_format is None else atom_format
+        ghost_format = "@{elem}" if ghost_format is None else ghost_format
+        umap = {"bohr": "! Bohrs", "angstrom": None}
```

Comment:
I suspect this `None` instead of `""` is going to cause trouble, too.

Reply:
Fixed this.